CHANGELOG.md

```diff
@@ -1,301 +1,119 @@
 # Change log
 
-The `next` version of 4.29 is now available. Planned release date is February 2024.
+The `next` version of 4.30 is now available. Planned release date is June 2024.
 
-![Current build version](https://img.shields.io/npm/v/arcgis-js-api/next?label=Current%20build)
-
-## Performance updates
-
-The [heatmapRendererCreator.createRenderer()](https://developers.arcgis.com/javascript/latest/api-reference/esri-smartMapping-renderers-heatmap.html#createRenderer) method for generating a [HeatmapRenderer](https://developers.arcgis.com/javascript/latest/api-reference/esri-renderers-HeatmapRenderer.html) is 100x faster than it was in version 4.28.
+![Current build version](https://img.shields.io/npm/v/@arcgis/core/next?label=Current%20build)
 
 ## Layer updates
 
-### Pagination on WFSLayer
-
-WFSLayer now uses pagination on requests to the service instead of requesting the features all at once, which should lead to better performance and load time of large WFSLayers. Use the `maxRecordCount` property to set the maximum number of features that can be returned in a single request.
-
-## MapView hitTest improvements
-
-### VectorTileLayer hitTest
-
-The [hitTest()](https://developers.arcgis.com/javascript/latest/api-reference/esri-views-MapView.html#hitTest) method on the MapView now returns features belonging to a [VectorTileLayer](https://developers.arcgis.com/javascript/latest/api-reference/esri-layers-VectorTileLayer.html) that intersect the hitTest point. This provides access to the attributes of those features, which can be useful for various purposes such as displaying additional information or performing further analysis.
-
-### MapView hitTest respects drawing order
-
-When performing a [`hitTest()`](https://developers.arcgis.com/javascript/latest/api-reference/esri-views-MapView.html#hitTest) on the MapView, the drawing order of the features in the layer is now respected. The returned features will adhere to the default drawing order of the data or a specific order if [`orderBy`](https://developers.arcgis.com/javascript/latest/api-reference/esri-layers-FeatureLayer.html#orderBy) is specified on the layer. This is a significant improvement from previous versions where the features were displayed based on the order they were returned from the server.
-
 ## Widget Updates
```

Comment:
Same for this one: I would probably just remove this or put TBD if it's going to be empty

Reply:
Ditto, and thanks!